[2025-03-19T19:07:57.919+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T19:07:59.604+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-12-07T00:00:00+00:00 [queued]>
[2025-03-19T19:08:00.064+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-12-07T00:00:00+00:00 [queued]>
[2025-03-19T19:08:00.084+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T19:08:00.655+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-12-07 00:00:00+00:00
[2025-03-19T19:08:00.826+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10147) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T19:08:00.849+0000] {standard_task_runner.py:72} INFO - Started process 10168 to run task
[2025-03-19T19:08:01.175+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'get_name', 'scheduled__2024-12-07T00:00:00+00:00', '--job-id', '2939', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmptcz_2vva']
[2025-03-19T19:08:01.262+0000] {standard_task_runner.py:105} INFO - Job 2939: Subtask get_name
[2025-03-19T19:08:02.213+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-12-07T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T19:08:10.617+0000] {job.py:229} INFO - Heartbeat recovered after 15.39 seconds
[2025-03-19T19:08:14.559+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-12-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-07T00:00:00+00:00'
[2025-03-19T19:08:14.687+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T19:08:14.810+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'John', 'last_name': 'Doe'}
[2025-03-19T19:08:16.814+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T19:08:17.576+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=get_name, run_id=scheduled__2024-12-07T00:00:00+00:00, execution_date=20241207T000000, start_date=20250319T190759, end_date=20250319T190817
[2025-03-19T19:08:20.085+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T19:08:20.577+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
